addons/event/migrations/9.0.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,127 @@
+---Fields in module 'event'---
+event        / event.event              / active (boolean)              : NEW
+# Nothing to do
+
+event        / event.event              / badge_front (html)            : NEW
+# Nothing to do
+
+event        / event.event              / color (integer)               : NEW
+# Nothing to do
+
+event        / event.event              / country_id (many2one)         : not a function anymore
+# Nothing to do as country_id is now a related field from address_id
+
+event        / event.event              / email_confirmation_id (many2one): DEL relation: email.template
+event        / event.event              / email_registration_id (many2one): DEL relation: email.template
```

Comment:
This worries me, as we have customers using specific templates for each event. How Odoo v9 handles this? Anyway, at least we need to keep the columns if an specific module restoring this function comes.